app/assets/javascripts/index/note.js

```diff
@@ -20,40 +19,41 @@ OSM.Note = function (map) {
       iconAnchor: [12, 40]
     })
   };
+  noteIcons.hidden = noteIcons.closed; // TODO replace with actual "hidden" icon when it's added
```

Comment:
Why do we want an icon for hidden notes? They are supposed to be hidden...

Reply:
Do you want to hide notes from moderators too or do you want to keep this bug?
https://github.com/openstreetmap/openstreetmap-website/issues/546
https://github.com/openstreetmap/openstreetmap-website/issues/1215

I fixed it here to check if other things are working. Of course I can put it back if you want.

I'm using the "closed" icon because I assume that the actual "hidden" icon, like introduced here https://github.com/openstreetmap/openstreetmap-website/issues/507 , still doesn't exist.